### DISCUSSION TITLE
AnsibleLookupError: Failed to access SSM parameter

### DISCUSSION THREAD

**Discussion by vijayavanisabbiti** [learndevopsonline/learndevopsonline#1680] [Q&A]
I'm getting the below error while trying to check the frontend or backend in server. 
Can you please help?

<img width="494" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/150422689/51a7fae7-e3f5-4d64-a2a3-2e72032c2f0f">

**Comment by sreedharm07** [comment]
can you turn on your instance

**Reply by vijayavanisabbiti** [reply]
it is running

**Reply by sreedharm07** [reply]
i tried many times its not logging in 
anyway this may be because of role attachment 
check weather you attached a role to the instance with admin permissions
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/c82edcf3-f6aa-4df6-8f09-577f101db162)
